roles/openshift_logging/defaults/main.yml

```diff
@@ -69,15 +69,15 @@ openshift_logging_kibana_ops_key: ""
 openshift_logging_kibana_ops_ca: ""
 
 openshift_logging_fluentd_nodeselector: "{{ openshift_hosted_logging_fluentd_nodeselector_label | default('logging-infra-fluentd=true') | map_from_pairs }}"
-openshift_logging_fluentd_cpu_limit: "{{ openshift_hosted_logging_fluentd_cpu_limit | default('100m') }}"
-openshift_logging_fluentd_memory_limit: "{{ openshift_hosted_logging_fluentd_memory_limit | default('512Mi') }}"
+openshift_logging_fluentd_cpu_limit: 100m
+openshift_logging_fluentd_memory_limit: 512Mi
```

Comment:
Indeed, I'm almost lost... :)  Do you think we still need to keep, e.g.,
openshift_logging_fluentd_cpu_limit: "{{ openshift_hosted_logging_fluentd_cpu_limit | default('100m') }}"
even if openshift_hosted_logging_fluentd_cpu_limit did not exist prior to this pr?  The cpu limit and memory limit were just hardcoded...